mlflow/java/client/src/main/java/org/mlflow/tracking/MlflowHttpCaller.java

```diff
@@ -128,6 +129,22 @@ String post(String path, String json) {
     }
   }
 
+  String patch(String path, String json) {
+    logger.debug("Sending PATCH " + path + ": " + json);
+    HttpPatch request = new HttpPatch();
+    fillRequestSettings(request, path);
```

Comment:
there's a lot of copypasta between get, post and patch (especially the latter two) - we could consolidate, but probably ok as is.

Reply:
I've created a new private function that encapsulates the common logic here.